requirements.txt

```diff
@@ -3,4 +3,5 @@ psutil>=5.8.0
 numpy>=1.19.5
 pandas>=1.1.5; platform.python_implementation == 'CPython'
 pandas<=1.2.5; platform.python_implementation == 'PyPy'
-dpkt>=1.9.7
\ No newline at end of file
+dpkt>=1.9.7
+scapy>=2.5.0
```

Comment:
Is it possible to use dpkt as it is faster and avoid us adding a dependy?

Reply:
Certainly, I'll be fixing that this week (most surely)

Reply:
Fixed, it is more than twice as fast!